deployment/mobile_apps/ios/README.md

```diff
@@ -0,0 +1,110 @@
+# Deployment - Mobile Apps - iOS
+
+**NOTE**: This assumes you are using ionic 2 but should translate pretty well
+for any cordova project. It also assumes you have Xcode set up appropriately
+and that you have the login credentials for the mohrlab dev Apple ID.
```

Comment:
Since this is a public guide, maybe we can make it slightly more general purpose? "login credentials for an Apple ID"?